src/redprl/judgment.fun

```diff
@@ -27,6 +27,7 @@ struct
            in
              (([],[]), tau)
            end
+       | MATCH_RECORD (lbl, _) => (([],[]), RedPrlSortData.EXP)
```

Comment:
@jonsterling I am fine with `RedPrlSortData.EXP` if there is no alias, but I was lost.